packages/lib-react-components/src/Tabs/theme.js

```diff
@@ -28,10 +28,14 @@ const theme = {
       const borderColor = getBorderColor(props)
       const activeTabHeaderColor = getActiveTabHeaderColor(props)
       const hoverBackgroundColor = getHoverBackground(props)
+      const tabChildren = props.children?.[0]?.props?.children
+      const flexBasis = tabChildren
+        ? `${100 / tabChildren.length}%`
```

Comment:
I'm confused. Why doesn't `props.children.length` work here?

Reply:
This looks a bit weird on phone screens, where flexbox moves the fourth tab down under the first three.

<img width="544" alt="The Tabs component with four tabs, in the storybook but set to iPhone 5. The fourth tab is rendered underneath the other three." src="https://user-images.githubusercontent.com/59547/176865736-c7868372-861a-4ddb-9d35-bfe20b836393.png">

Reply:
How does this look with the default Grommet tab styles?

Reply:
> I'm confused. Why doesn't `props.children.length` work here?

Turns out `props.children` always has exactly two children: the _tabs header,_ and the _tabs body_ (of the currently selected tab). So previously, whether you had 1 child tab or 100 child tab(s), each will always get allocated half the available flex space, i.e. `flex: 1 1 50%`

I'm assuming the _intent_ of `flex: 1 1 ${100 / props.children.length}%` was to ensure each child tab has an equal amount of flex-basis. (So with 3 tabs, each should get 33% base size.)

This does cause issues if we expect Tabs to have a lot of children with limited screen width, as you saw with the phone view.

> This looks a bit weird on phone screens, where flexbox moves the fourth tab down under the first three.
> How does this look with the default Grommet tab styles?

Grommet actually plays its much smarter by having [scrolling when there are too many child tab(s)](https://storybook.grommet.io/?path=/story/controls-tabs-responsive--responsive).

<img width="443" alt="image" src="https://user-images.githubusercontent.com/13952701/176905648-6ae5764f-5200-4311-a3e8-c3644f036302.png">

Not sure what we did to override this.